src/mc/mcsim.f03

```diff
@@ -129,6 +136,96 @@ pure function list_confinement()
               endif
           endif
 
+        ! set wlc_basepairs to prop if not slide move
+        if (MCTYPE/=13) then 
+            wlc_basepairs_prop = wlc_basepairs
+        endif
+
+         ! sterics check here !
+          left = minval(wlc_pointsMoved(1:wlc_nPointsMoved))
+          right = maxval(wlc_pointsMoved(1:wlc_nPointsMoved))
```

Comment:
This code here assumes the polymer is not in a ring.  This should either be fixed or a warning added that tells people too update this if they would like to do a ring.

Reply:
totally! i can put a warning that gets spit out if they set those parameters together in defines and ill also add documentation here